.gitignore

```diff
@@ -0,0 +1 @@
+/skietbaan_fe/src/components/ScoreCapture.css.orig
```

Comment:
Why that one specific and not all `*.orig`

Reply:
Because thats the only one. I think

Reply:
He meant that add for every .orig file.